common/app/common/commercial/hosted/HostedPage.scala

```diff
@@ -35,11 +36,24 @@ case class HostedCampaign(
   owner: String,
   logo: HostedLogo,
   cssClass: String,
-  brandColour: String,
-  brightFont: Boolean,
+  fontColour: FontColour,
   logoLink: Option[String] = None
 )
 
+case class FontColour(brandColour: String) {
+
+  lazy val shouldHaveDarkBackground = !isBright
+
+  lazy val isBright = {
+    val hexColour = brandColour.stripPrefix("#")
+    val rgb = Integer.parseInt(hexColour, 16)
+    val c = new Color(rgb)
+    val hsb = Color.RGBtoHSB(c.getRed, c.getGreen, c.getBlue, null)
+    val brightness = hsb(2)
+    brightness < 0.5
```

Comment:
I'd check to see if this 0.5 is the right amount. I think the colour contrast on those examples is too little, I would expect the text to be black not white.